std/meta.d

```diff
@@ -680,17 +680,25 @@ template staticMap(alias F, Args ...)
     assert(A == typeid(int));
 }
 
-/**
-Tests whether all given items satisfy a template predicate, i.e. evaluates to
-$(D F!(T[0]) && F!(T[1]) && ... && F!(T[$ - 1])).
+version (StdDdoc)
+{
+    /**
+       Tests whether all given items satisfy a template predicate, i.e. evaluates to
+       $(D F!(T[0]) && F!(T[1]) && ... && F!(T[$ - 1])).
 
-Evaluation is $(I not) short-circuited if a false result is encountered; the
-template predicate must be instantiable with all the given items.
- */
-template allSatisfy(alias F, T...)
+       Evaluation is $(I not) short-circuited if a false result is encountered; the
+       template predicate must be instantiable with all the given items.
+    */
+    template allSatisfy(alias F, T...)
```

Comment:
This `version(StdDdoc)` charade seems to grow out of control. I think we need to stop and look for a good solution to it.

Reply:
Can you think of any alternative solutions, @adamdruppe. What about adding support for overriding the template and function header declaration in the generated documentation?

Reply:
Note that `allSatisfy` and `anySatisfy` are currently the only two templates left that are aliases into `core.internal.traits`.

Reply:
Good to know, thanks.

Reply:
How bad does the ddoc look for the alias? I take it the parameters are not documented? If so, IMHO the best solution going forward is to fix that issue.

Reply:
Ping, @adamdruppe.

Reply:
> How bad does the ddoc look for the alias? I take it the parameters are not documented? If so, IMHO the best solution going forward is to fix that issue.

How can I test this?

Reply:
At least in the olden days you'd go in phobos and `make -j8 html`. That generates the documentation in a separate directory.

Reply:
`make -f posix.mak build-html` works for me on Linux.

Reply:
The no GC initialized thing at least has been for me due to a dmd-druntime version mismatch fuckup. adrdox when...

Reply:
Is this an advice on how to move forward, @maxhaton? If so, I don't grasp what your aiming at. Can you elaborate?